ui/src/components/WelcomeTutorial.tsx

```diff
@@ -0,0 +1,431 @@
+import * as React from "react";
+import styled from "styled-components";
+import Dialog from "@material-ui/core/Dialog";
+import { colors } from "../styles/colors";
+import Button from "@material-ui/core/Button";
+import IconButton from "@material-ui/core/IconButton";
+import CloseIcon from "@material-ui/icons/Close";
+import DialogTip from "../assets/images/white-arrow.png";
+import OverlayCircle from "../assets/images/small-clear-circle.svg";
+import BigOverlayCircle from "../assets/images/clear-circle.svg";
+import ShoeLogo from "../assets/images/welcome-shoe-logo.svg";
+import {
+  WelcomeTitleText,
+  WelcomeDescriptionText,
+  NavigateTitleText,
+  NavigateDescriptionText,
+} from "../styles/typography";
+
+const StyledArrowTip = styled.img`
+  position: absolute;
+  width: 29px;
+  height: 26px;
+  left: -5%;
+  top: 45%;
```

Comment:
Do we have no other way other than `position: absolute` + manually ajdusting element with css? Maybe with something like https://github.com/shipshapecode/react-shepherd?

Reply:
That looks pretty cool I wish I knew about it earlier! I know the manual adjustment is a bit suspect (esp the negative overflow) but it's relative + bounded to the dialog so it wouldn't cause issues with responsiveness and it seems to be the simplest solution? Could look at using/converting to that tour library in the future if we end up with a heavier tutorial ^^

Reply:
apparently this is a thing too https://docs.react-joyride.com/

Reply:
Good point, I guess it is only a 3 step tutorial. Let's use your approach for now then, and we can always change it later!